### PR TITLE
[New] Add v-mask@1.3.2 w/ git auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #9870
Related issue(s): # #

@pvnr0082t Please help me review this PR. Thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/probil/v-mask
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/v-mask
 * License and its reference: MIT, [ref](https://github.com/probil/v-mask/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 4
   - Count of stars: 30
   - Count of forks: 15
 * NPM download stats (optional):
   - Downloads in the last day: 18
   - Downloads in the last week: 111
   - Downloads in the last month: 489

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
@pvnr0082t After some search, this library seemed not used in the front end side. Let me ask in #9870.

**Comment by pvnr0082t** [comment]
@hare1039 You can also create an issue in https://github.com/probil/v-mask/issues.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by hare1039** [comment]
No updates currently

**Comment by sufuf3** [comment]
@hare1039 You could add the files from https://github.com/probil/v-mask/tree/v1.3.0/dist
Please continue this PR, thank you.

**Comment by extend1994** [comment]
ping @hare1039

**Comment by sufuf3** [comment]
@hare1039 
Would you mind working on this and helping improve this PR after final exam?
You could add the files from https://github.com/probil/v-mask/tree/v1.3.0/dist

Many thanks.

**Comment by trevorhreed** [comment]
@hare1039 This is how I intend to use this on the front end: https://rawgit.com/probil/v-mask--demo/master/dist/index.html

File: ajax/libs/v-mask/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/probil/v-mask.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```
Comment (extend1994) [inline ajax/libs/v-mask/package.json]: Let's exclude `v-mask.esm.js` which is for esm. See https://github.com/probil/v-mask/blob/master/build/rollup.conf.js#L18-L21

File: ajax/libs/v-mask/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "v-mask",
+  "filename": "v-mask.min.js",
+  "version": "1.3.2",
+  "description": "Simple input mask lib for vue.js",
```
Comment (extend1994) [inline ajax/libs/v-mask/package.json]: Please update description https://github.com/probil/v-mask/blob/master/package.json#L4

File: ajax/libs/v-mask/package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/probil/v-mask.git"
+  },
+  "keywords": [
+    "vue.js",
+    "vue",
+    "vue-mask"
+  ],
+  "author": {
```
Comment (extend1994) [inline ajax/libs/v-mask/package.json]: Please update author https://github.com/probil/v-mask/blob/master/package.json#L26

**Comment by sashberd** [comment]
@extend1994 The PR was updated. The .git ignore files was updated.

**Comment by sashberd** [comment]
@extend1994 I updated the PR. But I think as I also not an author of the PR and just help to cdnjs reduce number of the PR`s, such things as PR commit message also can be done by you in order to reduce PR time. Just a remark.

**Comment by PeterDaveHello** [comment]
@sashberd appreciate your help!

**Comment by extend1994** [comment]
@PeterDaveHello .gitignore excluded the wrong version. The ones should be excluded are v1.0.0 ~ v1.2.0. Could you help fix it? Thanks.

**Comment by sashberd** [comment]
@extend1994 @PeterDaveHello I created new [PR](https://github.com/cdnjs/cdnjs/pull/12805) in order to fix my fault. FYR and update

**Pull request merged**